server/org.eclipse.emfcloud.ecore.glsp/resources/enotation.ecore

```diff
@@ -3,19 +3,21 @@
     xmlns:ecore="http://www.eclipse.org/emf/2002/Ecore" name="enotation" nsURI="http://www.eclipsesource.com/glsp/ecore/notation"
     nsPrefix="enotation">
   <eClassifiers xsi:type="ecore:EClass" name="Shape" eSuperTypes="#//NotationElement">
-    <eStructuralFeatures xsi:type="ecore:EReference" name="position" eType="ecore:EClass  platform:/resource/org.eclipse.emfcloud.ecore.glsp/src/main/resources/glsp-graph.ecore#//GPoint"
+    <eStructuralFeatures xsi:type="ecore:EReference" name="position" eType="ecore:EClass ../src/main/resources/glsp-graph.ecore#//GPoint"
```

Comment:
These references look incorrect (But they also looked incorrect in the previous revision). It should be: `platform:/plugin/org.eclipse.glsp.graph/model/glsp-graph.ecore#//GPoint` (And similar for the URIs below). The Ecore Editor likes to use relative URIs, even when they don't make a lot of sense (Like when referencing external plug-ins), but I don't really understand why we reference anything from src/main/resources :confused:

Reply:
Ah thank you for spotting this! I used your proposed paths now.